modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/builder/ObjectTypeSchemaDelegate.java

```diff
@@ -368,7 +368,7 @@ private void declareObjectField(DslElementSyntax fieldDsl, ObjectFieldType field
       return;
     }
 
-    if (id.equals(Scheduler.class.getName())) {
+    if (id.equals(SchedulingStrategy.class.getName())) {
```

Comment:
you must keep both conditions. Existing extensions will still have Scheduler as their id